AWSLambdaAPI/Controllers/S3BucketController.cs

```diff
@@ -0,0 +1,130 @@
+﻿using Amazon.S3.Model;
+using Amazon.S3.Util;
+using Amazon.S3;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using NPOI.SS.Formula.Functions;
+using NPOI.SS.UserModel;
+using NPOI.XSSF.UserModel;
+using ExternalEntities.Misc;
+using Microsoft.Extensions.Options;
+using Repository.Interfaces;
+using Amazon.SecretsManager;
+using Amazon;
+using Amazon.SecretsManager.Model;
+
+namespace AWSLambdaAPI.Controllers
+{
+    [Route("[controller]")]
+    [ApiController]
+    public class S3BucketController : ControllerBase
+    {
+        private string bucketName = "task6.s3bucket";
```

Comment:
this should be a const variable with appropriate naming convention e.g BucketName

Reply:
Agree

Reply:
API project unloaded.